alternator/executor.cc

```diff
@@ -1629,7 +1665,7 @@ future<executor::request_return_type> rmw_operation::execute(service::storage_pr
             {timeout, std::move(permit), client_state, trace_state},
             db::consistency_level::LOCAL_SERIAL, db::consistency_level::LOCAL_QUORUM, timeout, timeout).then([this, read_command] (bool is_applied) mutable {
         if (!is_applied) {
-            return make_ready_future<executor::request_return_type>(api_error::conditional_check_failed("Failed condition."));
+            return make_ready_future<executor::request_return_type>(api_error::conditional_check_failed("The conditional request failed", std::move(_return_attributes)));
```

Comment:
I see now where you used the new "JSON" parameter, but still don't understand how it reaches the user - isn't this _json field just ignored by the server? I don't see a patch to server.cc's generate_error_reply() which prints this error.
I see you changed the error message itself a bit - is it because the new message fits DynamoDB's message better?

Reply:
yes, the message this is what dynamodb is currenty returning